server/src/interfaces.ts

```diff
@@ -30,4 +30,34 @@ export interface CustomerPayload {
  */
 export interface CustomerResponse extends CustomerPayload {
   id: number;
+  numCommits: number;
+}
+
+/**
+ * CommitStatus type contains the different states of a Commit.
+ */
+export type CommitStatus =
+  | 'ongoing'
+  | 'successful'
+  | 'paid'
+  | 'completed'
+  | 'unsuccessful';
+
+/**
+ * ListingPayload Interface that contains the fields of the payload that
```

Comment:
```suggestion
 * CommitPayload Interface that contains the fields of the payload that
```
You forgot to change this!

Reply:
oopsies!